changelog/security/2022-09-16-nvidia-drivers.md

```diff
@@ -0,0 +1 @@
+- nvidia-drivers (CVE-2022-31607, CVE-2022-31608, CVE-2022-31615)
```

Comment:
Link them to the NVD database, even if those are empty. Eventually they will show up.